source/extensions/http/custom_response/local_response_policy/local_response_policy.h

```diff
@@ -23,7 +23,7 @@ class LocalResponsePolicy : public Extensions::HttpFilters::CustomResponse::Poli
 public:
   LocalResponsePolicy(const envoy::extensions::http::custom_response::local_response_policy::v3::
                           LocalResponsePolicy& config,
-                      Server::Configuration::CommonFactoryContext& context);
+                      Server::Configuration::ServerFactoryContext& context);
```

Comment:
In general passing Generic makes more sense than server in case folks want (properly scoped) stats down the road WDYT?

Reply:
Actually, in previous implementation, the backend of `CommonFactoryContext` here will always be a `ServerFactoryContext.

As I said in the TODO, this potential bug is not introduced by this patch. The `LocalResponsePolicy` will be created in the `createPolicy` method which will always take a `ServerFactoryContext`.

This patch just wrappered the `ServerFactoryContext` to a `GenericFactoryContext` (for Formatter usage) and doesn't change logic or behavior.

```
Extensions::HttpFilters::CustomResponse::PolicySharedPtr
LocalResponseFactory::createPolicy(const Protobuf::Message& config,
                                   Envoy::Server::Configuration::ServerFactoryContext& context,
                                   Stats::StatName) {
  const auto& local_response_config =
      MessageUtil::downcastAndValidate<const LocalResponsePolicyProto&>(
          config, context.messageValidationVisitor());
  return std::make_shared<LocalResponsePolicy>(local_response_config, context);
}
```